### PR TITLE
Pragma to silence Visual Studio conversion warnings

### PR DESCRIPTION
**Pull request by tautschnig**
There are several cases of conversion from size_t to int that we cannot fix as
we use system library functions.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] My contribution is formatted in line with CODING_STANDARD.md.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 55917875ea7c30931a3a2d3d5a8f9b1b0548a321).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85741125

File: jbmc/src/java_bytecode/java_bytecode_parser.cpp [hunk 1]
```diff
@@ -705,7 +705,10 @@ void java_bytecode_parsert::rconstant_pool()
         std::string s;
         s.resize(bytes);
         for(std::string::iterator s_it=s.begin(); s_it!=s.end(); s_it++)
+#include <util/pragma_push.def>
```
Comment (peterschrammel) [inline jbmc/src/java_bytecode/java_bytecode_parser.cpp:708]: That's not quite readable. Maybe braces help?

**Review by peterschrammel — COMMENTED** [review]
This doesn't seem a practical solution. Is there no way to use an explicit cast to silence it? Or at least define a custom cast operator and use the pragma only there.

**Comment by tautschnig** [comment]
Let's clean up all the other warnings first. I'll revisit this once we have a manageable Visual Studio build log.

**Pull request closed without merging**